vlasovsolver/cpu_acc_load_blocks.cpp

```diff
@@ -165,7 +165,7 @@ cog.outl("#endif")
    if(dimension == 0) {
       for (vmesh::LocalID block_k=0; block_k<n_blocks; ++block_k) {
          Realf* __restrict__ data = blockContainer->getData(vmesh->getLocalID(blocks[block_k]));
-   #if (defined(VEC4D_AGNER) || defined(VEC4F_AGNER) || defined(VEC_FALLBACK_GENERIC)) && VECL == 4
+   #if (defined(VEC4D_AGNER) || defined(VEC4F_AGNER) || defined(VEC_FALLBACK_GENERIC) || defined(VEC4F_SVE) || defined(VEC4D_SVE)) && VECL == 4
```

Comment:
Note: This file is autogenerated by the cog code at the top. You now modified the generated code, but did *not* update the cog generator above!